tests/unit_tests/kernel_source_test.py

```diff
@@ -72,7 +73,8 @@ def test_get_module_from_source(source):
     llvm_file = os.path.join(source.kernel_dir, "net/core/utils.ll")
     if os.path.isfile(llvm_file):
         os.unlink(llvm_file)
-    mod = source.get_module_from_source("net/core/utils.c")
+    mod = source.get_module_from_source("net/core/utils.c",
+                                        datetime.datetime.now().timestamp())
```

Comment:
Could you also add a test where the source is not compiled because of a wrong timestamp? Something like, get current time, then touch the source file, and then try to compile it with the timestamp.

Reply:
I will try to add something like that.